### PR TITLE
[traceql] Fix missing event and link intrinsic handling in new vp5 metrics fetch layer

### PR DESCRIPTION
**Pull request by mdisibio**
**What this PR does**:
A query like `{ } | rate() by (event:name)` would incorrectly return nils when using vp5 and the new span-only fetch layer.

**Which issue(s) this PR fixes**:
Fixes #<issue number>

**Checklist**
- [x] Tests updated
- [ ] Documentation added
- [x] Changelog entry added under `.chloggen/` (run `make chlog-new`, or `make chlog-new FILENAME=<name>` to override the default branch-name file; see `.chloggen/README.md`)

### CODE REVIEW

File: tempodb/encoding/vparquet5/block_traceql_fetch.go [hunk 1]
```diff
@@ -1296,6 +1296,8 @@ func (c *spanCollector2) Collect(res *parquetquery.IteratorResult, param any) {
 				sp.addSpanAttr(newSpanAttr(e.Key), v)
 			case res.RowNumber[DefinitionLevelResourceSpans] >= 0:
 				sp.resourceAttrs = append(sp.resourceAttrs, attrVal{traceql.NewScopedAttribute(traceql.AttributeScopeResource, false, e.Key), v})
+			default:
+				panic("unhandled static value: " + e.Key)
```
Comment (mattdurham) [inline tempodb/encoding/vparquet5/block_traceql_fetch.go:1300]: Should this be a panic?

Comment (mdisibio) [reply tempodb/encoding/vparquet5/block_traceql_fetch.go:1300]: This might be kind of opinionated, but I've found them valuable for rooting out the unhandled edge cases.   99.99% this never triggers, our test suite already checks that all attributes of all columns of all scopes, etc, should be handled. But in case there is something out there, we need to know about it, and this makes it highly visible and also avoids returning silently bad data.  This isn't the only instance of panic, there are a few others.

**Review by mattdurham — DISMISSED** [review]

**Review by ie-pham — DISMISSED** [review]

**Review by mattdurham — APPROVED** [review]
